services/core-web/src/components/Forms/Securities/BondForm.js

```diff
@@ -71,6 +78,8 @@ export class BondForm extends Component {
     }));
   };
 
+  // TODO: this function will have to remove the file through a BE call
```

Comment:
I know we don't want to balloon scope here, but with other TODOs I've found throughout the system, by the time we get around to looking at them, they're 2-3 years old and many of the processes surrounding them have changed enough that they don't really help.  Maybe this would make more sense as a ticket for the backlog?  Or maybe both?